### PR TITLE
Write tutorial for DemoYesNoSUT.

### PR DESCRIPTION
**Pull request by brianwgoldman**
Also some minor tweaks to the code to make the demo better.

### CODE REVIEW

**Review by wpietri — APPROVED** [review]
A great step forward.

File: docs/tutorial_suts.md [hunk 1]
```diff
@@ -1,3 +1,93 @@
...
+
+```py
+class DemoYesNoSUTResponse(BaseModel):
+    number_of_words: int
+    text: str
+```
+
+Note that in this example inhert from [Pydantic](https://docs.pydantic.dev/latest/)'s `BaseModel` to aid in serialization and caching. This isn't strictly necessary, but we highly recommend doing so.
+
+To tell NewHELM what your native representations are, we us Python [Generics](https://mypy.readthedocs.io/en/stable/generics.html).
```
[the remark this answers is not in the document]
Comment (brianwgoldman) [reply docs/tutorial_suts.md]: Done.

File: docs/tutorial_suts.md [hunk 2]
```diff
@@ -1,3 +1,93 @@
 # SUT Demos
 
-TODO
\ No newline at end of file
+## Creating a basic SUT
+
+[Demo: DemoYesNoSUT](../demo_plugin/newhelm/suts/demo_01_yes_no_sut.py)
+
+Before diving into creating more realistic SUTs, lets start with with a toy example: A SUT that only answers the question "Does this prompt have an even number of words?"
```
[the remark this answers is not in the document]
Comment (brianwgoldman) [reply docs/tutorial_suts.md]: Fixed.

File: docs/tutorial_suts.md [hunk 3]
```diff
@@ -1,3 +1,93 @@
...
+    # Return Yes if the input is an even number of words
+    number_of_words = len(request.text.split())
+    answer = "Yes" if number_of_words % 2 == 0 else "No"
+    return DemoYesNoSUTResponse(number_of_words=number_of_words, text=answer)
+```
+
+Finally we convert the response back into a normalized form that Tests can understand:
+
+```py
+def translate_response(
```
[the remark this answers is not in the document]
Comment (brianwgoldman) [reply docs/tutorial_suts.md:66]: Good call out. This is an artifact of wanting to call it `Demo...SUT` and wanting the request to be named after its SUT. I renamed the objects to `DemoYesNoRequest` and `DemoYesNoResponse`.

**Pull request merged**